src/main/webapp/scripts/toggle.js

```diff
@@ -28,15 +36,12 @@ async function loadContentSection() {
     keywords.forEach(loadKeywordSection);
   }
   const address = encodeURI(getCookie('address'));
-  const location = getCookie('location');
   if (address != "") {
     loadCivicSectionFromAddress(address);
+  } else if (location != "") {
+    loadCivicSectionFromLocation(locationObj);
```

Comment:
If we always save the address as mentioned above we could load the civic section from address all the time.